airbyte-integrations/connectors/source-snowflake/unit_tests/test_schema_builder.py

```diff
@@ -0,0 +1,31 @@
+from source_snowflake.schema_builder import format_field
+from unittest.mock import MagicMock
+import pytest
+
+
+def test_format_field_semi_structured_type():
```

Comment:
Those tests are empty

Reply:
I am willing to add unittest from branch CDC. I cannot do it here because there are slight changes in the next PR